cns/NetworkContainerContract_test.go

```diff
@@ -56,18 +56,18 @@ func TestUnmarshalPodInfo(t *testing.T) {
 	}
 }
 
-func TestNewPodInfoFromIPConfigRequest(t *testing.T) {
+func TestNewPodInfoFromIPConfigsRequest(t *testing.T) {
```

Comment:
instead of changing the existing test (since the old code should still work) I would suggest adding a new test instead